models/issues/issue.go

```diff
@@ -127,6 +127,7 @@ type Issue struct {
 	PullRequest      *PullRequest     `xorm:"-"`
 	NumComments      int
 	Ref              string
+	Pin              int `xorm:"DEFAULT 0"`
```

Comment:
Could it be a two-word column name, eg: `PinOrder`?

`Pin` reads like "pinned" or not.